src/logo.tsx

```diff
@@ -0,0 +1,371 @@
+            </svg>
+        );
+    }
+
+    return (
+        <svg height={size} viewBox="0 0 72 72" fill={color}>
+            <path d="M55.8 52.2003H18V63.0003H55.8V52.2003Z" fill={colorSecondary} />
+            <path d="M38.3805 36.431H34.6798V31.658H38.3805C40.3136 31.658 41.0868 32.865 41.0868 34.0171C41.0868 35.2789 40.3136 36.431 38.3805 36.431ZM34.7351 19.0948H37.4966C40.2029 19.0948 40.0373 23.0447 37.4966 23.0447H34.7351V19.0948ZM49.1503 26.2267C52.1327 25.1844 53.7342 22.3864 53.7895 19.369C53.9552 13.7182 49.4819 9 43.8483 9H18V19.0398H21.6454V36.7602H18V46.8H43.7377C50.3654 46.8 55.5022 42.3013 55.7784 35.8276C56.0545 30.89 53.6792 27.9275 49.1503 26.2267Z" />
+        </svg>
+    );
+};
+
+type LogoBaseProps = {
+    skinName: SkinName;
+    size?: number;
+    type?: LogoType;
+};
+
+const LogoBase: React.FC<LogoBaseProps> = ({size = 48, skinName, type = 'isotype'}) => {
+    switch (skinName) {
+        case 'Movistar':
+            return <MovistarLogoImage size={size} type={type} />;
+        case 'Vivo':
+            return <VivoLogoImage size={size} type={type} />;
+        case 'O2':
+        case 'O2-classic':
+            return <O2LogoImage size={size} type={type} />;
+        case 'Telefonica':
+            return <TelefonicaLogoImage size={size} type={type} />;
+        case 'Blau':
+            return <BlauLogoImage size={size} type={type} />;
+        // todo WEB-761 what to do for unknown skinnames?
+        default:
+            return null;
+    }
+};
+type LogoPropsBase = {
+    size?: number;
+    type?: LogoType;
+    children?: void;
+};
+
+type LogoToProps = LogoPropsBase & {
+    trackingEvent?: TrackingEvent | ReadonlyArray<TrackingEvent>;
+    to: string;
+    fullPageOnWebView?: boolean;
+    replace?: boolean;
+};
+
+type LogoHrefProps = LogoPropsBase & {
+    trackingEvent?: TrackingEvent | ReadonlyArray<TrackingEvent>;
+    href: string;
+    newTab?: boolean;
+};
+
+type LogoOnPressProps = LogoPropsBase & {
+    trackingEvent?: TrackingEvent | ReadonlyArray<TrackingEvent>;
+    onPress: () => void;
+};
+
+type LogoProps = ExclusifyUnion<LogoPropsBase | LogoToProps | LogoHrefProps | LogoOnPressProps>;
+
+const MaybeTouchableLogo = (
+    logoTouchableProps: Omit<LogoProps, 'children'> & {children: JSX.Element}
+): JSX.Element => {
+    if (logoTouchableProps.to) {
+        return (
+            <Touchable
+                trackingEvent={logoTouchableProps.trackingEvent}
+                to={logoTouchableProps.to}
+                fullPageOnWebView={logoTouchableProps.fullPageOnWebView}
+                replace={logoTouchableProps.replace}
+                // todo WEB-761 how do we want to handle this?
+                aria-label={logoTouchableProps.to}
```

Comment:
how to handle this? letting pass it via prop?

Reply:
are we sure about this? I mean: the reader would spell the whole URL in place of some more intelligible